extensions/everhour/src/api/index.ts

```diff
@@ -16,27 +24,32 @@ const daysAgo = (days: number) => {
   return d;
 };
 
-export const getCurrentUser = async () => {
-  const response = await fetch("https://api.everhour.com/users/me", {
-    headers,
-  });
+// export const getCurrentUser = async () => {
+//   const response = await fetch("https://api.everhour.com/users/me", {
+//     headers,
+//   });
 
-  return (await response.json()) as any;
-};
+//   return (await response.json()) as any;
+// };
```

Comment:
if not needed then happy to get rid of this code, lets not leave it commented out